client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -71,25 +90,20 @@ export function getEnhancedTasks(
 				case 'plan_selected':
 					taskData = {
 						title: translate( 'Choose a Plan' ),
-						subtitle: displayGlobalStylesWarning
-							? translate(
-									'Your site contains custom colors that will only be visible once you upgrade to a Premium plan.'
-							  )
-							: '',
-						disabled: isVideoPressFlow( flow ),
+						subtitle: planWarningText,
 						actionDispatch: () => {
 							recordTaskClickTracksEvent( flow, task.completed, task.id );
 							const plansUrl = addQueryArgs( `/plans/${ siteSlug }`, {
-								...( displayGlobalStylesWarning && {
+								...( displayWarning && {
 									plan: PLAN_PREMIUM,
 									feature: FEATURE_ADVANCED_DESIGN_CUSTOMIZATION,
```

Comment:
This line should highlight the advanced design customisation row when clicking through to upgrade the plan. I'd guess you don't want that for videopress

![image](https://user-images.githubusercontent.com/1233880/206433155-d14f306c-13e3-4700-9ed3-19b0fba15012.png)

Reply:
Oh, thanks! Maybe I can highlight the `VideoPress Supported` row instead 👍

Reply:
Added in f99eac8c051fe607e4c4444d375798782e723bcc:

<img width="1095" alt="Screenshot 2022-12-12 at 9 23 49 AM" src="https://user-images.githubusercontent.com/789137/207101019-f762bd29-0535-44d2-a2db-91c6536c6476.png">